src/typescript/07-functions/functions.md

```diff
@@ -9,10 +9,11 @@
 
 In this part, we will:
 
-- Annotate functions parameters and return values.
-- Use optional parameters & rest parameters
+- Learn how to annotate functions parameters and return values
+- Learn how to use optional parameters & rest parameters
```

Comment:
I think this was better the way it was. See these pointers: https://bitovi.atlassian.net/wiki/spaces/BSG/pages/665976833/Bitovi+Academy+Style+Guide#Overview